### ISSUE TITLE
2025-02-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#75]
# Today's Arxiv Papers

(1) [Ola: Pushing the Frontiers of Omni-Modal Language Model with Progressive Modality Alignment](http://arxiv.org/abs/2502.04328v1), 2025-02-06

(2) [WorldSense: Evaluating Real-world Omnimodal Understanding for Multimodal LLMs](http://arxiv.org/abs/2502.04326v1), 2025-02-06

(3) [Speak Easy: Eliciting Harmful Jailbreaks from LLMs with Simple Interactions](http://arxiv.org/abs/2502.04322v1), 2025-02-06

(4) [ChamaleonLLM: Batch-Aware Dynamic Low-Rank Adaptation via Inference-Time Clusters](http://arxiv.org/abs/2502.04315v1), 2025-02-06

(5) [Great Models Think Alike and this Undermines AI Oversight](http://arxiv.org/abs/2502.04313v1), 2025-02-06

(6) [ScoreFlow: Mastering LLM Agent Workflows via Score-based Preference Optimization](http://arxiv.org/abs/2502.04306v1), 2025-02-06

(7) [Beyond Prompt Content: Enhancing LLM Performance via Content-Format Integrated Prompt Optimization](http://arxiv.org/abs/2502.04295v1), 2025-02-06

(8) [PILAF: Optimal Human Preference Sampling for Reward Modeling](http://arxiv.org/abs/2502.04270v1), 2025-02-06

(9) [Combining Language and App UI Analysis for the Automated Assessment of Bug Reproduction Steps](http://arxiv.org/abs/2502.04251v1), 2025-02-06

(10) [MAGA: MAssive Genre-Audience Reformulation to Pretraining Corpus Expansion](http://arxiv.org/abs/2502.04235v1), 2025-02-06

(11) [Can LLMs Hack Enterprise Networks? Autonomous Assumed Breach Penetration-Testing Active Directory Networks](http://arxiv.org/abs/2502.04227v1), 2025-02-06

(12) [Keep It Light! Simplifying Image Clustering Via Text-Free Adapters](http://arxiv.org/abs/2502.04226v1), 2025-02-06

(13) [Éclair -- Extracting Content and Layout with Integrated Reading Order for Documents](http://arxiv.org/abs/2502.04223v1), 2025-02-06

(14) [Sports and Women's Sports: Gender Bias in Text Generation with Olympic Data](http://arxiv.org/abs/2502.04218v1), 2025-02-06

(15) [Algorithmic causal structure emerging through compression](http://arxiv.org/abs/2502.04210v1), 2025-02-06

(16) ["Short-length" Adversarial Training Helps LLMs Defend "Long-length" Jailbreak Attacks: Theoretical and Empirical Evidence](http://arxiv.org/abs/2502.04204v1), 2025-02-06

(17) [The Best Instruction-Tuning Data are Those That Fit](http://arxiv.org/abs/2502.04194v1), 2025-02-06

(18) [PixFoundation: Are We Heading in the Right Direction with Pixel-level Vision Foundation Models?](http://arxiv.org/abs/2502.04192v1), 2025-02-06

(19) [Automated Microservice Pattern Instance Detection Using Infrastructure-as-Code Artifacts and Large Language Models](http://arxiv.org/abs/2502.04188v1), 2025-02-06

(20) [Are the Majority of Public Computational Notebooks Pathologically Non-Executable?](http://arxiv.org/abs/2502.04184v1), 2025-02-06

(21) [Multi-agent Architecture Search via Agentic Supernet](http://arxiv.org/abs/2502.04180v1), 2025-02-06

(22) [MRAMG-Bench: A BeyondText Benchmark for Multimodal Retrieval-Augmented Multimodal Generation](http://arxiv.org/abs/2502.04176v1), 2025-02-06

(23) [UltraIF: Advancing Instruction Following from the Wild](http://arxiv.org/abs/2502.04153v1), 2025-02-06

(24) [The Order Effect: Investigating Prompt Sensitivity in Closed-Source LLMs](http://arxiv.org/abs/2502.04134v1), 2025-02-06

(25) [Llasa: Scaling Train-Time and Inference-Time Compute for Llama-based Speech Synthesis](http://arxiv.org/abs/2502.04128v1), 2025-02-06

(26) [VTutor: An Open-Source SDK for Generative AI-Powered Animated Pedagogical Agents with Multi-Media Output](http://arxiv.org/abs/2502.04103v1), 2025-02-06

(27) [LLMs to Support a Domain Specific Knowledge Assistant](http://arxiv.org/abs/2502.04095v1), 2025-02-06

(28) [AttentionPredictor: Temporal Pattern Matters for Efficient LLM Inference](http://arxiv.org/abs/2502.04077v1), 2025-02-06

(29) [Controllable Emotion Generation with Emotion Vectors](http://arxiv.org/abs/2502.04075v1), 2025-02-06

(30) [Predicting Large Language Model Capabilities on Closed-Book QA Tasks Using Only Information Available Prior to Training](http://arxiv.org/abs/2502.04066v1), 2025-02-06

(31) [Leveraging Reasoning with Guidelines to Elicit and Utilize Knowledge for Enhancing Safety Alignment](http://arxiv.org/abs/2502.04040v1), 2025-02-06

(32) [Exploring Imbalanced Annotations for Effective In-Context Learning](http://arxiv.org/abs/2502.04037v1), 2025-02-06

(33) [Fine, I'll Merge It Myself: A Multi-Fidelity Framework for Automated Model Merging](http://arxiv.org/abs/2502.04030v1), 2025-02-06

(34) [Echo-Teddy: Preliminary Design and Development of Large Language Model-based Social Robot for Autistic Students](http://arxiv.org/abs/2502.04029v1), 2025-02-06

(35) [Quantification of Biodiversity from Historical Survey Text with LLM-based Best-Worst Scaling](http://arxiv.org/abs/2502.04022v1), 2025-02-06

(36) [Automating a Complete Software Test Process Using LLMs: An Automotive Case Study](http://arxiv.org/abs/2502.04008v1), 2025-02-06

(37) [CAD-Editor: A Locate-then-Infill Framework with Automated Training Data Synthesis for Text-Based CAD Editing](http://arxiv.org/abs/2502.03997v1), 2025-02-06

(38) [Ontology-Guided, Hybrid Prompt Learning for Generalization in Knowledge Graph Question Answering](http://arxiv.org/abs/2502.03992v1), 2025-02-06

(39) ["It Warned Me Just at the Right Moment": Exploring LLM-based Real-time Detection of Phone Scams](http://arxiv.org/abs/2502.03964v1), 2025-02-06

(40) [MAQInstruct: Instruction-based Unified Event Relation Extraction](http://arxiv.org/abs/2502.03954v1), 2025-02-06

(41) [Afrispeech-Dialog: A Benchmark Dataset for Spontaneous English Conversations in Healthcare and Beyond](http://arxiv.org/abs/2502.03945v1), 2025-02-06

(42) [Experiments with Large Language Models on Retrieval-Augmented Generation for Closed-Source Simulation Software](http://arxiv.org/abs/2502.03916v1), 2025-02-06

(43) [InfinitePOD: Building Datacenter-Scale High-Bandwidth Domain for LLM with Optical Circuit Switching Transceivers](http://arxiv.org/abs/2502.03885v1), 2025-02-06

(44) [Rank Also Matters: Hierarchical Configuration for Mixture of Adapter Experts in LLM Fine-Tuning](http://arxiv.org/abs/2502.03884v1), 2025-02-06

(45) [BOLT: Bootstrap Long Chain-of-Thought in Language Models without Distillation](http://arxiv.org/abs/2502.03860v1), 2025-02-06

(46) [Improving Natural Language Understanding for LLMs via Large-Scale Instruction Synthesis](http://arxiv.org/abs/2502.03843v1), 2025-02-06

(47) [FairT2I: Mitigating Social Bias in Text-to-Image Generation via Large Language Model-Assisted Detection and Attribute Rebalancing](http://arxiv.org/abs/2502.03826v1), 2025-02-06

(48) [Syntriever: How to Train Your Retriever with Synthetic Data from LLMs](http://arxiv.org/abs/2502.03824v1), 2025-02-06

(49) [PsyPlay: Personality-Infused Role-Playing Conversational Agents](http://arxiv.org/abs/2502.03821v1), 2025-02-06

(50) [Large Language Models for Multi-Robot Systems: A Survey](http://arxiv.org/abs/2502.03814v1), 2025-02-06

(51) [Identify Critical KV Cache in LLM Inference from an Output Perturbation Perspective](http://arxiv.org/abs/2502.03805v1), 2025-02-06

(52) [Understanding and Supporting Formal Email Exchange by Answering AI-Generated Questions](http://arxiv.org/abs/2502.03804v1), 2025-02-06

(53) [Enhancing Hallucination Detection through Noise Injection](http://arxiv.org/abs/2502.03799v1), 2025-02-06

(54) [It's All in The [MASK]: Simple Instruction-Tuning Enables BERT-like Masked Language Models As Generative Classifiers](http://arxiv.org/abs/2502.03793v1), 2025-02-06

(55) [Iterate to Accelerate: A Unified Framework for Iterative Reasoning and Feedback Convergence](http://arxiv.org/abs/2502.03787v1), 2025-02-06

(56) [GistVis: Automatic Generation of Word-scale Visualizations from Data-rich Documents](http://dx.doi.org/10.1145/3706598.3713881), 2025-02-06

(57) [Adaptive Semantic Prompt Caching with VectorQ](http://arxiv.org/abs/2502.03771v1), 2025-02-06

(58) [Hierarchical Contextual Manifold Alignment for Structuring Latent Representations in Large Language Models](http://arxiv.org/abs/2502.03766v1), 2025-02-06

(59) [Rethinking the Residual Distribution of Locate-then-Editing Methods in Model Editing](http://arxiv.org/abs/2502.03748v1), 2025-02-06

(60) [Speaking the Language of Teamwork: LLM-Guided Credit Assignment in Multi-Agent Reinforcement Learning](http://arxiv.org/abs/2502.03723v1), 2025-02-06

(61) [Efficiently Generating Expressive Quadruped Behaviors via Language-Guided Preference Learning](http://arxiv.org/abs/2502.03717v1), 2025-02-06

(62) [Boosting Knowledge Graph-based Recommendations through Confidence-Aware Augmentation with Large Language Models](http://arxiv.org/abs/2502.03715v1), 2025-02-06

(63) [MultiQ&A: An Analysis in Measuring Robustness via Automated Crowdsourcing of Question Perturbations and Answers](http://arxiv.org/abs/2502.03711v1), 2025-02-06

(64) [Aggregate and conquer: detecting and steering LLM concepts by combining nonlinear predictors over multiple layers](http://arxiv.org/abs/2502.03708v1), 2025-02-06

(65) [LLM Alignment as Retriever Optimization: An Information Retrieval Perspective](http://arxiv.org/abs/2502.03699v1), 2025-02-06

(66) [DocMIA: Document-Level Membership Inference Attacks against DocVQA Models](http://arxiv.org/abs/2502.03692v1), 2025-02-06

(67) [A Comparison of DeepSeek and Other LLMs](http://arxiv.org/abs/2502.03688v1), 2025-02-06

(68) [Controlled LLM Decoding via Discrete Auto-regressive Biasing](http://arxiv.org/abs/2502.03685v1), 2025-02-06

(69) [Reflection-Window Decoding: Text Generation with Selective Refinement](http://arxiv.org/abs/2502.03678v1), 2025-02-05

(70) [Advancing Reasoning in Large Language Models: Promising Methods and Approaches](http://arxiv.org/abs/2502.03671v1), 2025-02-05

(71) [Looking for the Inner Music: Probing LLMs' Understanding of Literary Style](http://arxiv.org/abs/2502.03647v1), 2025-02-05

(72) [Context-Preserving Gradient Modulation for Large Language Models: A Novel Approach to Semantic Consistency in Long-Form Text Generation](http://arxiv.org/abs/2502.03643v1), 2025-02-05

(73) [AdaPhish: AI-Powered Adaptive Defense and Education Resource Against Deceptive Emails](http://dx.doi.org/10.1109/ICAIC63015.2025.10848878), 2025-02-05

(74) [Bilevel ZOFO: Bridging Parameter-Efficient and Zeroth-Order Techniques for Efficient LLM Fine-Tuning and Meta-Training](http://arxiv.org/abs/2502.03604v1), 2025-02-05

(75) [HACK: Homomorphic Acceleration via Compression of the Key-Value Cache for Disaggregated LLM Inference](http://arxiv.org/abs/2502.03589v1), 2025-02-05

(76) [A Mixed-Methods Evaluation of LLM-Based Chatbots for Menopause](http://arxiv.org/abs/2502.03579v1), 2025-02-05

(77) [Chartist: Task-driven Eye Movement Control for Chart Reading](http://arxiv.org/abs/2502.03575v1), 2025-02-05

(78) [Code Simulation as a Proxy for High-order Tasks in Large Language Models](http://arxiv.org/abs/2502.03568v1), 2025-02-05

(79) [Kronecker Mask and Interpretive Prompts are Language-Action Video Learners](http://arxiv.org/abs/2502.03549v1), 2025-02-05

(80) [Do Large Language Model Benchmarks Test Reliability?](http://arxiv.org/abs/2502.03461v1), 2025-02-05

(81) [Adapt-Pruner: Adaptive Structural Pruning for Efficient Small Language Model Training](http://arxiv.org/abs/2502.03460v1), 2025-02-05

(82) [A Schema-Guided Reason-while-Retrieve framework for Reasoning on Scene Graphs with Large-Language-Models (LLMs)](http://arxiv.org/abs/2502.03450v1), 2025-02-05

(83) [YINYANG-ALIGN: Benchmarking Contradictory Objectives and Proposing Multi-Objective Optimization based DPO for Text-to-Image Alignment](http://arxiv.org/abs/2502.03512v1), 2025-02-05

(84) [Designing LLM-simulated Immersive Spaces to Enhance Autistic Children's Social Affordances Understanding](http://dx.doi.org/10.1145/3708359.3712142.), 2025-02-05

(85) [BFS-Prover: Scalable Best-First Tree Search for LLM-based Automatic Theorem Proving](http://arxiv.org/abs/2502.03438v1), 2025-02-05

(86) [On Fairness of Unified Multimodal Large Language Model for Image Generation](http://arxiv.org/abs/2502.03429v1), 2025-02-05

(87) [Harnessing Large Language Models for Curated Code Reviews](http://arxiv.org/abs/2502.03425v1), 2025-02-05

(88) [Think or Step-by-Step? UnZIPping the Black Box in Zero-Shot Prompts](http://arxiv.org/abs/2502.03418v1), 2025-02-05

(89) [An Empirical Exploration of ChatGPT's Ability to Support Problem Formulation Tasks for Mission Engineering and a Documentation of its Performance Variability](http://arxiv.org/abs/2502.03511v1), 2025-02-05

(90) [SPRI: Aligning Large Language Models with Context-Situated Principles](http://arxiv.org/abs/2502.03397v1), 2025-02-05

(91) [LIMO: Less is More for Reasoning](http://arxiv.org/abs/2502.03387v1), 2025-02-05

(92) [Demystifying Long Chain-of-Thought Reasoning in LLMs](http://arxiv.org/abs/2502.03373v1), 2025-02-05

(93) [PalimpChat: Declarative and Interactive AI analytics](http://arxiv.org/abs/2502.03368v1), 2025-02-05

(94) [Minerva: A Programmable Memory Test Benchmark for Language Models](http://arxiv.org/abs/2502.03358v1), 2025-02-05

(95) [ECM: A Unified Electronic Circuit Model for Explaining the Emergence of In-Context Learning and Chain-of-Thought in Large Language Model](http://arxiv.org/abs/2502.03325v1), 2025-02-05

(96) [Out-of-Distribution Detection using Synthetic Data Generation](http://arxiv.org/abs/2502.03323v1), 2025-02-05

(97) [Simplifying Formal Proof-Generating Models with ChatGPT and Basic Searching Techniques](http://arxiv.org/abs/2502.03321v1), 2025-02-05

(98) [Intent Representation Learning with Large Language Model for Recommendation](http://arxiv.org/abs/2502.03307v1), 2025-02-05

(99) [Harmony in Divergence: Towards Fast, Accurate, and Memory-efficient Zeroth-order LLM Fine-tuning](http://arxiv.org/abs/2502.03304v1), 2025-02-05

(100) [MeDiSumQA: Patient-Oriented Question-Answer Generation from Discharge Letters](http://arxiv.org/abs/2502.03298v1), 2025-02-05
